src/system/CategoriesModule/Controller/RegistryController.php

```diff
@@ -79,12 +79,32 @@ public function editAction(Request $request, CategoryRegistryEntity $registryEnt
 
     private function getCategorizableModules()
     {
-        $modules = $this->get('zikula_extensions_module.api.capability')->getExtensionsCapableOf(CapabilityApi::CATEGORIZABLE);
+        $entityManager = $this->get('doctrine')->getManager();
+
+        $modules = $entityManager->getRepository('ZikulaExtensionsModule:ExtensionEntity')
+            ->findBy(['state' => 3], ['displayname' => 'ASC']);
         $moduleOptions = [];
         foreach ($modules as $module) {
-            $moduleOptions[$module->getName()] = $module->getName();
+            $bundle = \ModUtil::getModule($module['name']);
+            if (null !== $bundle && !class_exists($bundle->getVersionClass())) {
+                // this check just confirming a Core-2.0 spec bundle - remove in 2.0.0
+                // then instead of getting MetaData, could just do $capabilityApi->getCapabilitiesOf($module['name'])
+                $capabilities = $bundle->getMetaData()->getCapabilities();
+                if (!isset($capabilities['categorizable'])) {
+                    continue; // skip this module if not categorizable
+                }
+            }
+            $moduleOptions[$module['name']] = $module['displayname'];
         }
 
+        /*
+            $modules = $this->get('zikula_extensions_module.api.capability')->getExtensionsCapableOf(CapabilityApi::CATEGORIZABLE);
+            $moduleOptions = [];
+            foreach ($modules as $module) {
+                $moduleOptions[$module->getName()] = $module->getName();
+            }
+        */
+
```

Comment:
Legacy modules like News do not specify a capability. Thus, the commented code does not include them. In 2.0 only the capability api should be used instead of the code which has been readded in this PR.